### PR TITLE
OSOE-128: Fix markdown files.

### PR DESCRIPTION
**Pull request by sarahelsaig**
OSOE-128

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -84,19 +81,20 @@ mkdir -p "$HOME/.local/bin"
 Also if you have installed NPM globally via your package manager, uninstall it to avoid future confusion.
 
 Next we install NVM and Node.js in userspace.
+
 1. Install NVM for your user: `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash`
-2. Type `nvm`. 
-    - If you get _nvm: command not found_ error, reload your shell profile first (`source ~/.bashrc`) and try again.
+2. Type `nvm`.
+    * If you get _nvm: command not found_ error, reload your shell profile first (`source ~/.bashrc`) and try again.
```
Comment (Piedone) [inline Readme.md]: `-` is easier to write for the bullet point and we use that everywhere.

File: Lombiq.Npm.Targets.targets [hunk 2]
```diff
@@ -48,12 +48,8 @@
     Inputs="@(NpmDotnetPrebuildWatchedFiles)"
     Outputs="$(NpmDotnetPrebuildStampFile)"
     Condition="'$(ExecDotnetPrebuildCommand)' == 'true' AND Exists($(PackageJson))">
-    <!-- Setting IgnoreStandardErrorWarningFormat="true" prevents MSBuild from setting the ExitCode to -1 when the
-      executed command actually returns 0, but logged warnings or errors during its execution. Refer to
-      https://docs.microsoft.com/en-us/visualstudio/msbuild/exec-task?view=vs-2022#remarks for more details. -->
     <Exec Command="$(NpmDotnetPrebuildCommand)"
           ConsoleToMSBuild="true"
-          IgnoreStandardErrorWarningFormat="true"
```
Comment (Piedone) [inline Lombiq.Npm.Targets.targets:60]: Why is this needed? Did you consult with Oliver?

**Review by Piedone — CHANGES_REQUESTED** [review]

File: Lombiq.Npm.Targets.targets (hunk 2 above)
Comment (sarahelsaig) [reply Lombiq.Npm.Targets.targets:60]: It must be removed, so we can emit MSBuild warnings. Discussion with Oliver [here](https://teams.microsoft.com/l/message/19:0e3ad2d722c74eef93386ecfeb790963@thread.skype/1658256414396?tenantId=959ede5a-38e9-44e6-adc8-85553beb8220&groupId=00c48e7c-aebf-4247-84cf-bb7c31061ef1&parentMessageId=1657013971481&teamName=Open%20Source%20Orchard%20Extensions%20developers&channelName=General&createdTime=1658256414396).

File: Readme.md (hunk 1 above)
Comment (sarahelsaig) [reply Readme.md]: We use both style very often, in fact `*` is used in this file.
This was an autofix as well. My guess is that the linter is trying to enforce consistent usage of bullet character within the document and just applies the first one it finds. In this case the [first one is an asterisk](https://github.com/Lombiq/NPM-Targets/blob/dev/Readme.md?plain=1#L62)  (link points to file dev branch).

Comment (sarahelsaig) [reply Readme.md]: Also, `*` is <kbd>AltGr</kbd>+<kbd>-</kbd>.
I agree that typing only <kbd>-</kbd> is easier,, but not by so much that we need to worry about it.

Comment (0liver) [reply Readme.md]: Asterisks are prettier.

Hyphens are easier to type.

Two-dimensional optimization problem ahead. ⚠️ 

Once we agree on "the best" bullet point character, we can enforce it using [configuration](https://github.com/DavidAnson/markdownlint/blob/v0.26.0/doc/Rules.md#md004---unordered-list-style).

File: Lombiq.Npm.Targets.targets (hunk 2 above)
Comment (Piedone) [reply Lombiq.Npm.Targets.targets:60]: OK then.

File: Readme.md (hunk 1 above)
Comment (Piedone) [reply Readme.md]: This file using asterisk before is an exception, then, because we overwhelmingly use hyphen otherwise, please use that here too.

Comment (sarahelsaig) [reply Readme.md]: fwiw all of the license.md files use asterisk too.

**Review by Piedone — APPROVED** [review]

**Review by 0liver — APPROVED** [review]

**Pull request merged**